custom/icds/messaging/indicators.py

```diff
@@ -108,6 +127,37 @@ def location_names_from_user_id(self, user_ids):
         return {self.awc_locations[l] for l in loc_ids}
 
 
+class AWWAggregatePerformanceIndicator(AWWIndicator):
+    template = 'ls_aggregate_performance.txt'
```

Comment:
we'll need a different template for the aww performance report since some of the headings are different (like timely visits vs. home visits)